main.go

```diff
@@ -46,61 +40,43 @@ type silenceResponse struct {
 }
 
 func main() {
-	clusterInit, err := getClusterCreationTime()
```

Comment:
Is there a reason why we're getting rid of this cluster timestamp check?  I'm not sure off the top of my head how MCC pushes out changes to this job, so we'd want to confirm that it's not going to re-run the job on existing clusters when we push an update.

If it doesn't re-run the job on update, then we're safe to remove this, however if it does re-run the job this could silence an already-ready production cluster if it's experiencing a blip of something updating.

Reply:
Yeah, I struggled with that. You're probably right that we should keep this check in place -- but from what @jharrington22 was telling me yesterday, one hour may be way too short. James, what do you think?

Reply:
The job _should_ be deployed within one hour via MCC, but really I'd be comfortable putting this to even the first 4 hours or so that a cluster is deployed, that would make testing easier and a bit more flexible.

Reply:
Roger wilco, I'll do that. Mebbe I'll add support for an env var so we can configure it (including for testing).

Reply:
This is done via 060f06b.